tck/src/test/java/org/reactivestreams/tck/flow/support/TCKVerificationSupport.java

```diff
@@ -81,6 +81,18 @@ public void requireTestSkip(ThrowingRunnable run, String msgPart) {
     throw new RuntimeException("Expected TCK to SKIP this test, instead if PASSed!");
   }
 
+  public void requireOptionalTestPass(ThrowingRunnable run) {
+    try {
+      run.run();
+    } catch (SkipException skip) {
+      throw new RuntimeException("Expected TCK to PASS this test, instead it was SKIPPED", skip.getCause());
+    } catch (Throwable throwable) {
+      throw new RuntimeException(
```

Comment:
Repackage Errors as Exceptions?

Reply:
Just being consistent with `requireTestFailure` and `requireTestSkip`.

Reply:
Does not really matter much as those will end up failing the test in the suite and one may always look into the logs. It's not like it's a production app where we have to watch out to not accidentally continue after an OOM or something... Fine as is I think